### ISSUE TITLE
Getters/setters

### ISSUE THREAD

**Issue by domenic** [jackbsteinberg/get-originals-rewriter#26]
Getters/setters are similar to methods; the main difference is that they're a type of property access expression that's not a call expression. Also, being sure to distinguish getter vs. setter cases.

Inputs:

```js
const set = new Set();
set.size;

const url = new URL();
url.href;
url.pathname = "foo";
```

Outputs, ignoring the constructor transforms:

```js
import { apply as Reflect_apply } from "std:global/Reflect";
import { size_get as Set_size_get } from "std:global/Set";
import { href_get as URL_href_get, pathname_set as URL_pathname_set } from "std:global/URL";

const set = new Set();
Reflect_apply(Set_size_get, set); // it's OK to pass [] as last arg, just redundant

const url = new URL();
Reflect_apply(URL_href_get, url);
Reflect_apply(URL_pathname_set, url, ["foo"]);
```

Note: the above examples dodge #25, we'll also want getter/setter tests for those.

**Comment by domenic** [comment]
Important note: this should only apply to accessor properties, not data properties. Notably, `array.length` should not get rewritten at all; it should stay `array.length`.

I'm not sure if there's a more generic detection story. We may end up just special-casing `array.length` as its one of the only data properties in the standard library.

**Comment by domenic** [comment]
I've concluded TypeScript does not the information to tell whether something's an accessor or data property. We should hard-code in array and string length for now, and have a follow-up to handle other cases.

The long-term solution is still probably a hard-coded list. Only the JS spec uses data properties, and it's pretty small relative to the entirety of the web. It's also not in a machine-scrapable format. So we'll just have to have someone (probably me) go through the JS spec and find all the data properties. I would guess there are probably like 5-10 max (with the 2 `length`s being by far the most common).

**Comment by domenic** [comment]
Created #33 to track the above-discussed detection of data properties, so that this issue can focus on the core functionality of accessor replacement (with only the two length properties hard-coded as data properties for now).

**Comment by jackbsteinberg** [comment]
Generic getter / setter case resolved in #28, closing this issue
